src/CookieCrumble/src/CookieCrumble/CookieCrumble.csproj

```diff
@@ -15,6 +15,7 @@
     <PackageReference Include="Microsoft.AspNetCore.WebUtilities" Version="2.2.0" />
     <PackageReference Include="Xunit" Version="2.4.1" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
+    <PackageReference Include="MSTest.TestFramework" Version="3.0.4" />
```

Comment:
I do not want to have mstest referenced in all of our tests ... we probably should abstract this more and move things around a bit. I will think on this a bit.

Reply:
I did think about splitting it out into separate assemblies for each test framework - so one shared assembly with most of the logic, then one for XUnit, one for MSTest etc (a bit like how the Snapshot test assembly was previously)

Reply:
Refactored the code in a follow-up PR (https://github.com/ChilliCream/graphql-platform/pull/6537) so we can close this one